### PR TITLE
AP_Scripting: Add getting of gyro/accel values

### PR DESCRIPTION
**Pull request by haydendonald**
Added bindings for ins:get_gyro(instance) and ins:get_accel(instance) so that scripting can get the readings from individual sensors. We need this to be able to validate that the IMUs are reading correctly for testing purposes.

This has been tested using the Cube Orange Plus

### CODE REVIEW

**Review by tridge — APPROVED** [review]

File: libraries/AP_Scripting/generator/description/bindings.desc [hunk 1]
```diff
@@ -38,7 +38,7 @@ singleton AP_AHRS method get_gyro Vector3f
 singleton AP_AHRS method get_accel Vector3f
 singleton AP_AHRS method get_hagl boolean float'Null
 singleton AP_AHRS method wind_estimate Vector3f
-singleton AP_AHRS method wind_alignment float'skip_check float'skip_check
+singleton AP_AHRS method wind_alignment floagt'skip_check float'skip_check
```
Comment (amilcarlucas) [inline libraries/AP_Scripting/generator/description/bindings.desc:41]: Is this intentional?

**Review by peterbarker — CHANGES_REQUESTED** [review]
As Amilcar points out - that doesn't look right

File: libraries/AP_Scripting/generator/description/bindings.desc (hunk 1 above)
Comment (haydendonald) [reply libraries/AP_Scripting/generator/description/bindings.desc]: Very much not. Fixed it now :)

**Comment by haydendonald** [comment]
> As Amilcar points out - that doesn't look right

Fixed :)

**Pull request merged**